src/ocaml/typing/short_paths.ml

```diff
@@ -1743,6 +1743,7 @@ module Basis = struct
       depends : string list;
       alias_depends : string list;
       desc : Desc.Module.t;
+      visibility : Desc.module_visibility;
```

Comment:
Do you actually need to add a new field to be threaded around here? IIRC the only effect of the `deprecated` field is to treat the module as hidden. Perhaps you could just rename the `deprecated` field to `visibility` instead?

Reply:
I suppose not. I left it in to make it easier if we ever want to do something different. But that would be easy to undo, so I'll remove it.